notebooks/ICESat-2_Cloud_Access/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -53,7 +53,7 @@
     "### **Example of end product** \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %time
</pre><p>For demostration, using <code>%time</code> is helpful to show how quick this access mode is - but maybe not as quick as we would like.  However, as a tutorial is timing the operation necessary? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I was questioning that as well. I don't think it is necessary to include in a tutorial unless we are specifically teaching something about the time of that request or how/when/why to use %time.</p>

Reply:
I have removed the %time